src/common/tusb_fifo.c

```diff
@@ -556,15 +576,15 @@ void tu_fifo_correct_read_pointer(tu_fifo_t* f)
 /******************************************************************************/
 bool tu_fifo_read(tu_fifo_t* f, void * buffer)
 {
-  tu_fifo_lock(f);                                          // TODO: Here we may distinguish for read and write pointer mutexes!
+  tu_fifo_lock_rd(f);                                          // TODO: Here we may distinguish for read and write pointer mutexes!
```

Comment:
We could remove the `// TODO` now :smiley:

Reply:
Done! :)